lib/MetaCPAN/Web/Controller/About.pm

```diff
@@ -5,6 +5,16 @@ use Format::Human::Bytes;
 
 BEGIN { extends 'MetaCPAN::Web::Controller' }
 
+sub auto : Private {
+    my ( $self, $c ) = @_;
+
+    $c->add_surrogate_key('about');
+    $c->res->header(
+        'Cache-Control' => 'max-age=' . $c->cdn_times->{one_day} );
+    $c->cdn_cache_ttl( $c->cdn_times->{one_day} );
```

Comment:
I wondered about that, but 'Cache-Control' isn't specifically cdn functionality, it's just another header, that said I might review it again some time once I've done some more complex examples